packages/notifi-react-example/src/NotifiCard/NotifiCard.tsx

```diff
@@ -39,11 +38,11 @@ export const NotifiCard: React.FC = () => {
     },
   };
 
-  const intercomInputSeparators: NotifiInputSeparators = {
-    emailSeparator: {
-      content: 'OR',
-    },
-  };
+  // const intercomInputSeparators: NotifiInputSeparators = {
```

Comment:
We should probably not commit any tests to `notifi-react-example`. Hmm, we should probably add `notifi-react-example` to `.gitignore` at some point.